tests/integration/test_coverage.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+import subprocess
+
+from coverage_comment import coverage
+from coverage_comment import subprocess as subprocess_module
+
+
+def test_get_added_lines(
+    commit, file_path, integration_dir, in_integration_env, write_file
+):
+    """
+    Lines added in the base_ref should not appear as added in HEAD
+    """
+    git = subprocess_module.Git()
+    relative_file_path = file_path.relative_to(integration_dir)
+
+    def _check_added_lines():
+        added_lines = coverage.get_added_lines(git, "main")
+
+        assert added_lines == {
+            relative_file_path: list(range(7, 13))  # Line numbers start at 1
+        }
```

Comment:
I'd rather add 2 duplicated lines inside the test than an inner function (so as to make tests as easy to read as possible)

Also, I believe it's ok if you make it a single line:
```python
assert coverage.get_added_lines(git, "main") == {
    relative_file_path: list(range(7, 13))  # Line numbers start at 1
}
```
though that's your choice :)